roadmap/implementers-guide/src/runtime/router.md

```diff
@@ -75,6 +75,14 @@ struct HrmpChannel {
     /// The total size in bytes of all message payloads in the channel.
     /// Invariant: should be less or equal to `limit_used_bytes`.
     used_bytes: u32,
+    /// A head of the Message Queue Chain for this channel. Each link in this chain has a form:
+	/// `(prev_head, B, H(M))`, where
+	/// - `prev_head`: is the previous value of `mqc_head`.
+	/// - `B`: is the [relay-chain] block number in which a message was appended
+	/// - `H(M)`: is the hash of the message being appended.
+	/// This value is initialized to a special value that consists of all zeroes which indicates
```

Comment:
Is this comment about the value of the field when no messages are pending?

Reply:
No, this comment refers to the initial state assigned to this field when a new fresh channel is created. One can answer a question if there are pending messages in the channel only having:
- the mqc head
- the receiver's watermark
- the preimage of the latest link (to extract the block number of the latest message)
FWIW, this is similar to what we have in XCMP